### DISCUSSION TITLE
Hardware Transcoding with Nvidia in Docker Compose

### DISCUSSION THREAD

**Discussion by DeltaTango69** [photoprism/photoprism#4033] [Q&A]
According to the help, in order to enable hardware transcoding using the Nvidia GPU, you should install the Nvidia Container toolkit. However, it's not entirely clear to me based on the instructions how I should install it in Docker Compose or Photoprism?! Are there no instructions for dummies?

https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html

**Comment by dotfortun3-code** [comment]
In the official documentation, it shows both of these things: 
https://docs.photoprism.app/getting-started/advanced/transcoding/#nvidia-container-toolkit

You need to install the Nvidia Container Toolkit on the host machine, not in Docker or PhotoPrism. Assuming you are using Linux as your host, you just install it using a package manager: 
https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html#installing-with-apt

Then following the instructions in the first link to update the docker compose to allow PhotoPrism to use the GPU.
